src/test/resources/xform_tests/real_form_with_cycle_errors.xml

```diff
@@ -0,0 +1,7060 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<h:html xmlns:h="http://www.w3.org/1999/xhtml" xmlns:orx="http://openrosa.org/jr/xforms" xmlns="http://www.w3.org/2002/xforms" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:jr="http://openrosa.org/javarosa" xmlns:vellum="http://commcarehq.org/xforms/vellum">
```

Comment:
I wonder if it's possible to use a smaller form for the test case that can repro the issue. Current one is 7k lines of xml which is not ideal.

Reply:
Hey, agreed that this seems like a big form. It also adds more time to tests than I'm comfortable with, but I couldn't think of a way to specifically clean out only these question elements and retain the "Regression." 

I'm not sure if it actually matters much to have the regression test for the future if it ends up causing issues. i'll mull it over  a bit